tests/test_keys.py

```diff
@@ -0,0 +1,47 @@
+import pytest
+
+from textual.app import App
+from textual.keys import _character_to_key
+
+
+@pytest.mark.parametrize(
+    "character,key",
+    [
+        ("1", "1"),
+        ("2", "2"),
+        ("a", "a"),
+        ("z", "z"),
+        ("_", "underscore"),
+        (" ", "space"),
+        ("~", "tilde"),
+        ("?", "question_mark"),
+        ("£", "pound_sign"),
+        (",", "comma"),
+    ],
+)
+def test_character_to_key(character: str, key: str) -> None:
+    assert _character_to_key(character) == key
+
+
+async def test_character_bindings():
```

Comment:
Should we include the `ctrl+char` case in one of these tests or is that covered elsewhere?

Reply:
THere's no equivalent singe character for that though.